packages/host/src/components/Navigation/RequireAuthentication.tsx

```diff
@@ -1,20 +1,24 @@
-import React, { FC } from 'react';
+import React, { FC, useEffect } from 'react';
 import {
-  Navigate,
   useAuthContext,
   useLocation,
+  useNavigate,
 } from '@openmsupply-client/common';
 import { AppRoute } from '@openmsupply-client/config';
 
 export const RequireAuthentication: FC = ({ children }) => {
   const { token } = useAuthContext();
   const location = useLocation();
+  const navigate = useNavigate();
 
-  if (!token) {
-    return (
-      <Navigate to={`/${AppRoute.Login}`} state={{ from: location }} replace />
-    );
-  }
+  useEffect(() => {
```

Comment:
changed to an effect, with no deps, so that it only gets called on load. this way, the logged out alert can be shown - otherwise, this will redirect and the alert never gets displayed 🙄